packages/create-plugin/src/utils/utils.prettifyFiles.ts

```diff
@@ -19,3 +22,12 @@ export async function prettifyFiles(exportPath: string) {
   }
   return 'Successfully ran prettier against new plugin.';
 }
+
+export function isPrettierUsed() {
+  return Boolean(getPrettierVersion());
+}
+
+function getPrettierVersion() {
+  const packageJson = getPackageJson();
+  return packageJson.devDependencies?.prettier || packageJson.dependencies?.prettier;
```

Comment:
I was slightly concerned that using a semver range with npx (e.g. `npx prettier@^2.8.7 . --write`) would cause it to fail but it turns out it works! 🥳